src/currencyWallets/api.js

```diff
@@ -43,7 +43,7 @@ const fakeMetadata = {
   notes: ''
 }
 
-const PRECISION = 1 / 1000000000 // 0.000 000 001
+export const PRECISION = '1'
```

Comment:
We shouldn't need `PRECISION` anymore - the max spend should always be absolutely precise, down to the very last Satoshi or Wei.